bindings/gumjs/runtime/hexdump.js

```diff
@@ -59,8 +61,9 @@ function hexdump(target, options) {
     if (bufferOffset !== 0)
       result.push('\n');
 
+    let address = (startAddress + offset).toString(16);
     result.push(
-      offsetColor, pad(offset.toString(16), leftColumnWidth, '0'), resetColor,
+      offsetColor, pad(address, leftColumnWidth, '0'), resetColor,
```

Comment:
Should be possible to simplify this to just:

```suggestion
      offsetColor, pad(startAddress.add(offset).toString(16), leftColumnWidth, '0'), resetColor,
```

Reply:
Agreed!